.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -17,6 +17,8 @@ https://github.com/microsoft/pylance-release/blob/master/TROUBLESHOOTING.md#fili
 <!--
 To find your version:
 
+Open the VS Code extensions panel. Select Pylance from the list of installed extensions. The version appears in the page header at the top right.
+
 Select "View: Toggle Output" from the command palette (Ctrl+Shift+P on Windows/Linux, Command+Shift+P on macOS), then select "Python Language Server" in the dropdown on the right. Look for the line Pylance Language Server version X in the console.
```

Comment:
Do we need this line if the extension's menu gives it?

Reply:
I'm adding this based on feedback where user didn't know where to get the version from (since Pylance wasn't starting).

Reply:
what's the extension menu, is that different from the extensions panel?

Reply:
Same thing.

I guess my point was that if the version was there in the panel with the extension's name, we wouldn't need to ask them to open the output channel to grab it. But, it should be opened anyway to get logs, so...

Reply:
oh do you mean get it from the left side rather than the top?
![image](https://user-images.githubusercontent.com/1696845/87484292-a89b7180-c5ea-11ea-8d35-1110d41297d8.png)

Reply:
Both are equivalent (the left is easier), but my comment is about the part of the template that still says to consult the output window for the printed Pylance version.

Reply:
in troubleshooting I made it more obvious it's "either" one, I can do that here too

Reply:
Yes, works for me.

Reply:
Updated